recipes/sortmerna/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sortmerna" %}
+{% set version = "4.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6b62def30704ea956e1de060b602f9bebc8f2ba68107c59329b332500997b1d2
+
+build:
+  number: 0
+  run_exports: []
+  #skip: True  # [osx]
```

Comment:
If you want `run_exports` to keep parity with Bioconda standards, then use them; otherwise, tidy up.

```suggestion
```